csm/plugins/cortx/rgw.py

```diff
@@ -85,12 +102,44 @@ def _build_response(self, operation, response) -> Any:
                 mapped_response = parsed_response_payload.load()
                 RGWPlugin._params_cleanup(mapped_response)
             except Exception as e:
-                Log.error(f"Error occured while coverting raw api response to required response. {e}")
+                Log.error(f"Error occured while coverting raw api response to\
+                    required response: {e}")
                 raise CsmInternalError(const.S3_CLIENT_ERROR_MSG)
         return mapped_response
 
+    def _supress_response_keys(self, operation, response) -> Any:
+        """
+        Remove specific keys from response based on an operation.
+
+        Args:
+            operation (str): IAM operation.
+            response (dict): Response from s3 server.
+
+        Raises:
+            CsmInternalError
+
+        Returns:
+            odified response.
+        """
+        suppressed_response = response
+        keys = self._api_suppress_payload_schema.get(operation)
```

Comment:
if operation is not present then it will throw error,
either handled exception or default value for get operation and check

Reply:
@abhijit1patil, 
if operation is not present, default None will be returned. And in that case original response will be returned back.